pkg/api/validation/schema.go

```diff
@@ -88,6 +101,29 @@ func (s *SwaggerSchema) ValidateBytes(data []byte) error {
 	if kind == nil {
 		return fmt.Errorf("kind not set")
 	}
+	if kind.(string) == "List" {
+		items, err := unpackObject(obj.(map[string]interface{}))
+		if err != nil {
+			return err
+		}
+		for _, item := range items {
+			fields := item.(map[string]interface{})
```

Comment:
@smarterclayton Thanks. I had not found runtime.Unstructured before. You are right. I do this because of it is not runtime.Object.